src/test/java/spoon/test/ctElement/CtElementImplTest.java

```diff
@@ -0,0 +1,41 @@
+package spoon.test.ctElement;
+
+import org.junit.jupiter.api.Test;
+import spoon.Launcher;
+import spoon.reflect.declaration.CtAnnotation;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.factory.Factory;
+import spoon.support.compiler.FileSystemFile;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CtElementImplTest {
+
+    @Test
+    void testRemoveAnnotation() {
+        // contract: removeAnnotation returns true after removing an annotation of a class containing a single
+        // annotation, and returns false when a non existing annotation is tried to be removed
+
+        // arrange
+        Launcher spoon = new Launcher();
+        spoon.addInputResource(new FileSystemFile("./src/test/java/spoon/test/ctElement/CtElementImplTest.java"));
+        spoon.buildModel();
+        Factory factory = spoon.getFactory();
+
+        CtClass<Annotated> annotatedClass = factory.Class().get(Annotated.class);
```

Comment:
We don't typically add resources inside of the actual test file, as that forces Spoon to parse a lot more stuff than just the intended resource. Arguably, it was a mistake by me to suggest it for the template tests.

Anyway, here your test resource is so small that you can just add it as inline text.

```suggestion
        CtClass<?> annotatedClass = Launcher.parseClass("@SuppressWarnings(\"unchecked\") class Annotated { }");
```